mesosutil/constants.go

```diff
@@ -2,5 +2,5 @@ package mesosutil
 
 const (
 	// MesosVersion indicates the supported mesos version.
-	MesosVersion = "0.23.0"
```

Comment:
No. This is the version that is supported in this git ref. If you want others, vendor with another tag.